src/Auth0.ManagementApi/Clients/UsersClient.cs

```diff
@@ -22,6 +22,16 @@ public UsersClient(ApiConnection connection)
         {
         }
 
+        /// <inheritdoc />
+        public Task AssignRolesAsync(string id, AssignRolesRequest request)
+        {
+            return Connection.PostAsync<AssignUsersRequest>("users/{id}/roles", request, null, null,
```

Comment:
AssignUsersRequest => AssignRolesRequest?

Reply:
Absolutely!  Thanks for that.  Fixed.